src/Core/Core.js

```diff
@@ -11841,7 +11863,10 @@ export default class Core {
       return;
     }
 
-    console.log("result from saving to database:", resp.data);
+    console.log(
+      `result from saving to database ${this.itemNumber}:`,
+      resp.data,
+    );
```

Comment:
I would leave this out or replace it why `console.debug`

Reply:
removed